sycl/include/CL/__spirv/spirv_ops.hpp

```diff
@@ -123,6 +123,14 @@ template <typename T, std::size_t R, std::size_t C,
 extern SYCL_EXTERNAL size_t __spirv_JointMatrixWorkItemLengthINTEL(
     JOINT_MATRIX_INTEL(T, R, C, L, S, U) *);
 
+template <typename T, std::size_t R, std::size_t C,
+          __spv::MatrixUse U = __spv::MatrixUse::Unnecessary,
+          __spv::MatrixLayout L = __spv::MatrixLayout::RowMajor,
+          __spv::Scope::Flag S = __spv::Scope::Flag::Subgroup>
+extern SYCL_EXTERNAL std::tuple<T, T>
+__spirv_JointMatrixWorkItemElemCoord(JOINT_MATRIX_INTEL(T, R, C, L, S, U) *,
```

Comment:
There is no such thing as std::tuple in SPIR-V. The instruction should return `int2` and if we want to create a tuple for get_coord API, then we should read elements from this vector to create tuple.

Reply:
Thanks for the comment.  Can you please tell a bit more about this `int2` type?  Is there any documentation/ code that I can take a look?

Reply:
It's a 2 elements vector. int2 is a spelling from OpenCL, but guess the appropriate alias should be known for DPCPP, see: https://registry.khronos.org/SYCL/specs/sycl-2020/html/sycl-2020.html#_aliases

Reply:
Done

Reply:
eh, why we can't use sycl::vec here?